docs/pages/world/tables.mdx

```diff
@@ -20,6 +21,16 @@ When a `System` reads or writes storage via [table libraries](/store/table-libra
 
 <Illustration />
 
+<Callout type="warning" emoji="⚠️">
+  Once a table is created its schema should be considered *immutable*. You could, in theory, modify the schema by
+  changing the content of the `store:Tables` table, but doing so corrupts the table and confuses the synchronization
+  code, which expects the table's metadata to be consistent.
```

Comment:
I don't know if we should draw attention to "these could _technically_ be modified but you're gonna break things"